vendor/engines/saml_authentication/README.md

```diff
@@ -0,0 +1,80 @@
+# SamlAuthentication Engine
+
+NUcore uses [Devise](https://github.com/plataformatec/devise) for authentication,
+and uses the [`saml_authenticatable`](https://github.com/apokalipto/devise_saml_authenticatable)
+module for SAML integration.
+
+## Enabling SAML Authentication
+
+Add the following line to your `Gemfile` if it is not already present:
+
+```ruby
+gem "saml_authentication", path: "vendor/engines/saml_authentication"`
+```
+
+In `config/settings.yml`, uncomment the `saml` section and adjust the settings
+as necessary to connect with your Identity Provider (IdP).
+
+```yaml
+saml:
+  idp_metadata: "https://websso.example.com/idp/metadata"
+  certificate_file: path/to/file.p12
+  attribute_map:
+    "PersonImmutableID": "username"
+    "User.email": "email"
+    "User.FirstName": "first_name"
+    "User.LastName": "last_name"
+```
+
+* `idp_metadata`: The URL for your IdP's metadata. This should be provided to you by the IdP.
+* `certificate_file` (Optional): A `.p12` certificate file for signing your requests.
+  _Do not check this in to version control_
+* `attribute_map`: A mapping from the IdP's attributes to the `User`'s attributes.
```

Comment:
Can you be a little more specific that the `User` attributes are referring to the nucore db table?